src/components/Table/index.jsx

```diff
@@ -1,8 +1,14 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './style.css';
+import StorageData from '../../utils/LocalStorage';
 
 const Table = ({ data }) => {
-  const handelDelete = () => {};
+  const handelDelete = (i) => {
+    console.log(i);
+    data.splice(i, 1);
```

Comment:
this function is ok if you work in dom but in our case, every change in data should re-render the component so this is react thing